src/reynir_correct/settings.py

```diff
@@ -620,13 +555,22 @@ def get_code(wrong_form: str) -> str:
             code = Ritmyndir.get_malsnid(wrong_form)
         if not code:
             code = R_EINKUNN[Ritmyndir.get_eink(wrong_form)]
+        if not code:
+            code = "R001"
         return code
 
+    @staticmethod
+    def get_entry(wrong_form: str, index: int) -> Any:
+        entry = Ritmyndir.DICT.get(
+            wrong_form, Ritmyndir.DICT.get(wrong_form.lower(), None)
```

Comment:
The `None` default is strictly speaking not needed, it is implicit

Reply:
Done